src/libstd/keyword_docs.rs

```diff
@@ -925,8 +925,14 @@ mod mut_keyword {}
 //
 /// Make an item visible to others.
 ///
-/// The documentation for this keyword is [not yet complete]. Pull requests welcome!
-///
+/// The keyword `pub` makes any module, function, or data structure accessible from inside
+/// of external modules. The `pub` keyword may also be used in a `use` declaration to re-export
+/// an identifier from a namespace.
+///
+/// For more information on the `pub` keyword, please see the [visibility] section
+/// of the Reference. Or the [visibility] section of Rust By Example.
+/// [visibility]: ../reference/visibility-and-privacy.html?highlight=pub#visibility-and-privacy
+/// [visibility]:../rust-by-example/mod/visibility.html
```

Comment:
These will need to be distinct identifiers; one option is to reword the above to be: `please see the visibility sections of the [Reference] and examples in the [Rust by Example] collection.`.